gapii/cc/spy.h

```diff
@@ -73,6 +71,8 @@ class Spy : public GlesSpy, public GvrSpy, public VulkanSpy {
                         const gvr_buffer_viewport_list* list,
                         gvr_mat4_abi head_space_from_start_space);
 
+  bool checkEndTrace() override;
```

Comment:
We never check the return value of this, so we should not be returning anything, or we should be checking it in all of the call-sites.

Reply:
Removed return value